packages/app/components/Tile/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import styled, { css } from "styled-components/native";
+import { Theme } from "../../provider/theme/theme.config";
+import Card from "../../../../apps/next/src/components/Card";
+
+const Value = styled.Text`
+  font-weight: 800;
+  font-size: 36px;
+  color: ${({ theme }: { theme: Theme }) => theme.colors.text};
+
+  ${({ theme }: { theme: Theme }) =>
+    theme.getBreakPoint?.({
+      xl: css`
+        font-size: 42px;
+        line-height: 60px;
+        ${({ theme }) =>
+          theme.styleFor({
+            web: css`
+              margin-bottom: 32px;
+            `,
+            native: css`
+              margin-bottom: ${theme.scale(32)}px;
+            `,
+          })}
+      `,
+      lg: css`
+        font-size: 36px;
+        line-height: 56px;
+        ${({ theme }) =>
+          theme.styleFor({
+            web: css`
+              margin-bottom: 30px;
+            `,
+            native: css`
+              margin-bottom: ${theme.scale(30)}px;
+            `,
+          })}
+      `,
+      md: css`
+        font-size: 36px;
+        ${({ theme }) =>
+          theme.styleFor({
+            web: css`
+              margin-bottom: 20px;
+            `,
+            native: css`
+              margin-bottom: ${theme.scale(20)}px;
+            `,
+          })}
+      `,
+      sm: css`
+        font-size: 36px;
+        ${({ theme }) =>
+          theme.styleFor({
+            web: css`
+              margin-bottom: 10px;
+            `,
+            native: css`
+              margin-bottom: ${theme.scale(10)}px;
+            `,
+          })}
+      `,
+    })}
+`;
+
+const Text = styled.Text`
+  color: ${({ theme }: { theme: Theme }) => theme.colors.text};
+  font-weight: 500;
+  font-size: 12px;
+  text-align: center;
+
+  ${({ theme }: { theme: Theme }) =>
+    theme.getBreakPoint?.({
+      lg: css`
+        font-size: 16px;
+      `,
+    })}
+`;
+
+const TileCard = styled(Card)<{ spaced?: boolean; theme: Theme }>`
+  width: 225px;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  ${({ theme }) =>
```

Comment:
this is a breakpoint, that will be used only in **web** version, please move it in **styleFor** for **web**